web3/_utils/method_formatters.py

```diff
@@ -498,9 +498,9 @@ def get_request_formatters(
     method_name: Union[RPCEndpoint, Callable[..., RPCEndpoint]]
 ) -> Dict[str, Callable[..., Any]]:
     request_formatter_maps = (
-        METHOD_NORMALIZERS,
         ABI_REQUEST_FORMATTERS,
         PYTHONIC_REQUEST_FORMATTERS,
+        METHOD_NORMALIZERS,
```

Comment:
channeling an old review: if this order is important, might be worth documenting with a comment explanation.